src/wmain.c

```diff
@@ -0,0 +1,43 @@
+#include <windows.h>
+
+// https://docs.microsoft.com/en-us/cpp/build/reference/entry-entry-point-symbol
+#pragma comment(linker, "/ENTRY:wWinMainCRTStartup")
```

Comment:
I would discourage you from using `pragma comment(linker....`.  It's "hidden magic".  I would prefer to see this explicitly declared in the CMake code.